frontend/src/scenes/web-analytics/WebAnalyticsFilters.tsx

```diff
@@ -235,3 +277,116 @@ export const WebAnalyticsCompareFilter = (): JSX.Element | null => {
 
     return <CompareFilter compareFilter={compareFilter} updateCompareFilter={setCompareFilter} />
 }
+
+const ShareButton = (): JSX.Element => {
+    const handleShare = (): void => {
+        void copyToClipboard(window.location.href, 'link')
+    }
+
+    return (
+        <LemonButton
+            type="secondary"
+            size="small"
+            icon={<IconLink />}
+            tooltip="Share"
+            tooltipPlacement="top"
+            onClick={handleShare}
+            data-attr="web-analytics-share-button"
+        />
+    )
+}
+
+function FiltersPopover(): JSX.Element {
+    const [displayFilters, setDisplayFilters] = useState(false)
+    const { rawWebAnalyticsFilters, isPathCleaningEnabled, conversionGoal, preAggregatedEnabled, productTab } =
+        useValues(webAnalyticsLogic)
+
+    const { setWebAnalyticsFilters, setIsPathCleaningEnabled, setConversionGoal } = useActions(webAnalyticsLogic)
+    const { featureFlags } = useValues(featureFlagLogic)
+
+    const { hasAvailableFeature } = useValues(userLogic)
+
+    const hasAdvancedPaths = hasAvailableFeature(AvailableFeature.PATHS_ADVANCED)
+    const showConversionGoal =
+        productTab === ProductTab.ANALYTICS &&
+        (!preAggregatedEnabled || featureFlags[FEATURE_FLAGS.WEB_ANALYTICS_CONVERSION_GOAL_PREAGG])
+
+    const taxonomicGroupTypes = getWebAnalyticsTaxonomicGroupTypes(preAggregatedEnabled ?? false)
+    const propertyAllowList = preAggregatedEnabled ? WEB_ANALYTICS_PROPERTY_ALLOW_LIST : undefined
+
+    const activeFilterCount = rawWebAnalyticsFilters.length + (conversionGoal ? 1 : 0) + (isPathCleaningEnabled ? 1 : 0)
```

Comment:
I think path cleaning should not count as a "filter". It is more of a general setting, and it will (likely) always be on, so having the orange circle there might make people ignore it since it will always be there. (I also worry about this for the Posthog AI being confused for having filters enabled)

For future improvements, I wonder if should have a row in the botton showing the applied filters instead of just the button there. https://posthog.slack.com/archives/C05LJK1N3CP/p1765567454191339?thread_ts=1765481986.952139&cid=C05LJK1N3CP